java-sdk/radar-schemas-core/src/main/java/org/radarbase/schema/specification/active/questionnaire/QuestionnaireDataTopic.java

```diff
@@ -38,6 +38,6 @@ public URL getQuestionnaireDefinitionUrl() {
     @Override
     protected void propertiesMap(Map<String, Object> props, boolean reduced) {
         super.propertiesMap(props, reduced);
-        props.put("questionnaireDefinitionUrl", questionnaireDefinitionUrl);
+        props.put("questionnaire_definition_url", questionnaireDefinitionUrl);
```

Comment:
do we have to write the specifications for adding new questionnaires after using this version? @mpgxvii

Reply:
I checked the questionnaire code and this variable is not used. The new definition mimics the style of the other variables.